cmd/awscollector/Dockerfile

```diff
@@ -72,6 +75,7 @@ COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certifica
 COPY --from=package /workspace/awscollector /awscollector
 COPY --from=package /workspace/config/ /etc/
 COPY --from=package /workspace/healthcheck /healthcheck
+COPY --from=ecstarget ecs_sd_targets.yaml /etc/
```

Comment:
can you detail how adding an empty file here will create an workaround? also can you add comments describing what is the workaround and a link to the upstream issue?

your idea is to make this test work only in the dev branch?

Reply:
by comments I mean that you should add comments in the dockerfile explaining that this line is a workaround and when it could be removed.

Reply:
yeah I am currently not sure if there is going to be any fix for this, probably the solution that we have?

Reply:
Sure, but what is special about copying an empty file in a docker image? can you add comments in the dockerfile, since this is a workaround?